css/main.css

```diff
@@ -86,22 +94,18 @@ kbd {
 
 
 .topbar{
-	width: 100%;
-	position: fixed;
-	background-color: var(--base-color);
-	top: 0; /* hug the top of the window instead of having a buffer */
-	left: 0;
-	z-index: 1;/*Lets topbar render ontop of iframe*/
-}
-
-.topbarNav{
-	padding: 0.3em;
 	display: flex;
 	flex-wrap: nowrap;
 	justify-content: space-evenly;
 	align-items: stretch;
-	box-sizing: border-box;
+	width: 100%;
 	max-height: 1.8em; /* 2*padding + text height + a teeny bit of padding*/
+	background-color: var(--base-color);
+	padding: 0.3em;
+	box-sizing: border-box;
+	top: 0; /* hug the top of the window instead of having a buffer */
+	left: 0;
```

Comment:
I don't think we need a top or left since we aren't using a position style. This would be good if it was sticky or some other position, but in this case, not needed.

Reply:
fixed